src/braket/circuits/gates.py

```diff
@@ -1831,7 +1862,13 @@ def fixed_qubit_count() -> int:
 
     @staticmethod
     @circuit.subroutine(register=True)
-    def ccnot(control1: QubitInput, control2: QubitInput, target: QubitInput) -> Instruction:
+    def ccnot(
+        control1: QubitInput,
+        control2: QubitInput,
+        target: QubitInput,
+        *,
+        control: Optional[QubitSetInput] = None,
```

Comment:
Not sure if I like this this, but I also don't have a better solution off the top of my head.

Reply:
Yup same